src/home_robot/home_robot/agent/objectnav_agent/objectnav_agent.py

```diff
@@ -280,7 +280,9 @@ def act(self, obs: Observations) -> Tuple[DiscreteNavigationAction, Dict[str, An
         elif self.timesteps[0] > self.max_steps:
             action = DiscreteNavigationAction.STOP
         else:
-            action, closest_goal_map = self.planner.plan(**planner_inputs[0])
+            action, closest_goal_map = self.planner.plan(
+                **planner_inputs[0], stg_dilated=True
```

Comment:
Can we make this part of the agent __init__ as well?

Reply:
Fixed this